src/components/button/test-button.js

```diff
@@ -30,6 +30,43 @@ describe('Button', function() {
     };
   });
 
+  it('default matches the snapshot', function() {
+    const wrapper = renderComponent();
+    expect(wrapper).toMatchSnapshot();
+    expect(wrapper.find('.p-button').length).toEqual(1);
+  });
+
+  it('neutral matches the snapshot', function() {
+    const wrapper = renderComponent({
+      modifier: 'neutral'
+    });
+    expect(wrapper).toMatchSnapshot();
+    expect(wrapper.find('.p-button--neutral').length).toEqual(1);
+  });
+
+  it('positive matches the snapshot', function() {
+    const wrapper = renderComponent({
+      modifier: 'positive'
+    });
+    expect(wrapper).toMatchSnapshot();
+    expect(wrapper.find('.p-button--positive').length).toEqual(1);
+  });
+
+  it('negative matches the snapshot', function() {
+    const wrapper = renderComponent({
+      modifier: 'negative'
+    });
+    expect(wrapper).toMatchSnapshot();
+    expect(wrapper.find('.p-button--negative').length).toEqual(1);
+  });
+
+  it('with custom type matches snapshot', function() {
```

Comment:
Do the modifiers work on the various types? Can you also add tests for these please.

Reply:
I don't see the added tests for this query.